### PR TITLE
Update create_metadata_account

### PR DESCRIPTION
**Pull request by juchiast**

### CODE REVIEW

File: crates/cmds-solana/src/nft/create_metadata_account.rs [hunk 1]
```diff
@@ -1,183 +1,113 @@
...
+                    .map(|c| space_wrapper::state::creator::Creator {
+                        key: c.address,
+                        verified: c.verified.unwrap_or(false),
+                        share: c.share,
+                    })
+                    .collect(),
+                seller_fee_basis_points: inputs.metadata.seller_fee_basis_points,
+                collection: inputs.collection_mint_account,
+            }
+            .data(),
```
Comment (juchiast) [inline crates/cmds-solana/src/nft/create_metadata_account.rs:101]: @enzotar I think this is enough for the instruction data? Previously, there is this part after the instruction data:

https://github.com/space-operator/node-registry/blob/f6f9d6401f7a4342d4b0f2bfe05031b83b7e11a9/crates/cmds-solana/src/nft/create_metadata_account.rs#L39-L56 

Any reason why it is added?

File: crates/cmds-solana/src/nft/create_metadata_account.rs [hunk 2]
```diff
@@ -189,7 +119,9 @@ pub enum Input {
 }
 
 #[derive(Serialize, Deserialize, Debug)]
-pub struct InputStruct {
+pub struct Input {
+    #[serde(flatten)]
+    pub update_authority: UpdateAuthority,
```
Comment (juchiast) [inline crates/cmds-solana/src/nft/create_metadata_account.rs:124]: @enzotar you can use `#[serde(flatten)]` for this

File: crates/cmds-solana/src/nft/create_metadata_account.rs [hunk 3]
```diff
@@ -200,8 +132,8 @@ pub struct InputStruct {
     pub metadata: NftMetadata,
     pub metadata_uri: String,
     pub uses: NftUses,
-    // #[serde(with = "value::pubkey::opt")]
-    // pub collection_mint_account: Option<Pubkey>,
```
Comment (juchiast) [inline crates/cmds-solana/src/nft/create_metadata_account.rs:204]: Did you encounter any problem with this previously?

Comment (enzotar) [reply crates/cmds-solana/src/nft/create_metadata_account.rs:204]: The opt; can't get it to work

File: crates/cmds-solana/src/nft/create_metadata_account.rs (hunk 2 above)
Comment (enzotar) [reply crates/cmds-solana/src/nft/create_metadata_account.rs:124]: Ok

File: crates/cmds-solana/src/nft/create_metadata_account.rs (hunk 1 above)
Comment (enzotar) [reply crates/cmds-solana/src/nft/create_metadata_account.rs:101]: Ok. Not sure. Think maybe rushed to make it work

File: crates/cmds-solana/src/nft/create_metadata_account.rs (hunk 3 above)
Comment (juchiast) [reply crates/cmds-solana/src/nft/create_metadata_account.rs:204]: Fixed

**Pull request merged**